client/src/components/Forms/FamilyIncome/Form.tsx

```
@@ -93,6 +96,11 @@ export const FamilyIncomeForm: React.FC<FamilyIncomeFormProps> = ({
     );
   }
 
+  const { obj: child, setErrorsHidden } = useValidationErrors<Child>(
```

does this actually work for all forms? Because lookng at this hook, it only seems to set `validationErrors` to undefined on the top-level object (i.e. the child) and so i would expect any already-distributed-by-the-backend errors to still exist on the enrollments/fundings/family/income determinations. I wonder if there's not some simpler way to do this where we create a context that holds a flag like `shouldDisplayFormErrors` and then make all form fields use that flag to decide if they should display an error or not. 

Wouldn't consider that ^ blocking if this does indeed work, but if we do need to do deeper removal of `validationErrors` on these objects to effectively hide all errors in the front end, then we'd prob want to retain two copies of the object (one with errors, one without) and return the appropriate one given the flag as opposed to recreating the error-less object each time

---

Yeah I guess it's time to actually refactor that-- will make it a context that passes back status funcs